Dockerfile

```diff
@@ -28,9 +28,9 @@ RUN apt-get update && apt-get -y install \
 
 # Build and install CMake from source.
 WORKDIR /usr/src
-RUN git clone git://cmake.org/cmake.git CMake && \
+RUN git clone --branch v3.4.3 --depth 1 \
+     git://cmake.org/cmake.git CMake && \
```

Comment:
This OK for using tagged versions, but it not work if we want to check out a specific sha on master, e.g. when we want to switch to master after the additional CMAKE_CROSSCOMPILING_EMULATOR patches are merged.